doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -36,9 +38,17 @@ lower_case_table_names=0
 
 #### Install / Configure Nginx
 
-`apt-get install php7.0-cli php7.0-mysql php7.0-gd php7.0-snmp php-pear php7.0-curl php7.0-fpm snmp graphviz php7.0-mcrypt php7.0-json nginx-full fping imagemagick whois mtr-tiny nmap python-mysqldb snmpd php-net-ipv4 php-net-ipv6 rrdtool git`
+`sudo apt-get install php7.0-cli php7.0-mysql php7.0-gd php7.0-snmp php-pear php7.0-curl php7.0-fpm snmp graphviz php7.0-mcrypt php7.0-json nginx-full fping imagemagick whois mtr-tiny nmap python-mysqldb snmpd php-net-ipv4 php-net-ipv6 rrdtool git curl`
```

Comment:
Please remove sudo. The opening paragraph states these commands are to be run as root, adding a mixture of sudo / non-sudo is confusing.